addon-test-support/setup-page.js

```diff
@@ -0,0 +1,11 @@
+import { create } from './create';
+
+/**
+ * Sets up a reference to `this.page` with the correct context set
+ */
+export default function(hooks, page) {
+  hooks.beforeEach(function() {
+    this.page = create(page);
+    this.page.setContext(this);
```

Comment:
What is the purpose of `setContext`? I think it's only applicable for the `moduleForIntegration` tests, but that doesn't seem a case here, cause we use use `setupTest` thing?

Reply:
I think I've got it. You just avoid this annoying `setContext` thing.
I think it enforces all the pages which are instantiated with this helper to switch to an integration mode which makes it unusable in the other kind of tests.

Reply:
We're currently only using this helper for integration tests, since acceptance tests don't need the page object to be created and the context set.

I was under the impression that it was required for you to use the `create` and `setContext` APIs when using page objects in integration tests, and this little helper just made the setup easier. Is that not the case?

Reply:
`moduleForIntegration` is the "old" version of the `ember-qunit` API -- `setupRenderingTest` is the "new" version of the same context, so using `setContext` on the page object is required, as far as I know.

Reply:
Yes, you are right, `setContext` is required to enable page objects within `moduleForIntegration` tests.

However there is a [support of new test helpers](https://github.com/san650/ember-cli-page-object/pull/357) included in the beta of ember-cli-page-object. I you aware of that? Is there any  reason you want to keep using `moduleFor`?

Reply:
As far as I understand, this way it's possible to use the latest test-helpers while keep using Ember jquery events.

There few issues with `setContext`:
- It's limited to legacy Ember integration mode only.
- It has to be set and removed for each page object instance.

I think we need a public way to explicitly specify mode the enable, smth like:

```js
import { setupPageObjects } from 'ember-cli-page-object/test-support';
import { EmberJqueryAdapter } from 'ember-cli-page-object/adapters'

module('...', function(hooks)
  setupPageObjects(hooks, {
    adapter: EmberJqueryAdapter
  });
```

Going to write some issues for this soon.